### ISSUE TITLE
ArgumentError (unknown keyword: tracer)

### ISSUE THREAD

**Issue by thedanielhanke** [rmosolgo/graphql-libgraphqlparser-ruby#26]
using graphql-gem v 1.7.6 (mac) GraphQL.parse fails throwing the named ArgumentError.

brute-forcing it to work by ignoring parameters except for the query:

```ruby
GraphQL.define_singleton_method :parse do |query, *_|
  parse_with_libgraphqlparser(query)
end
```

not even close to a solution, but does its job until its officially fixed.
cheers

**Comment by rmosolgo** [comment]
Thanks for reporting this! I forgot to update this gem after adding `GraphQL::Tracing` 😬

**Comment by rmosolgo** [comment]
(And, glad you found a workaround, thanks for sharing it!)

**Comment by rmosolgo** [comment]
Just released 1.2.0 with a fix, thanks again for reporting this issue!
